src/operator/constants.tsx

```diff
@@ -0,0 +1,289 @@
+    name: 'last-name',
+    defaultValue: '',
+  },
+  {
+    path: 'onboardingState',
+    header: 'Onboarding State',
+    name: 'onboarding-state',
+    defaultValue: '',
+  },
+  {
+    path: 'sfdcContactId',
+    header: 'SFDC Contact ID',
+    name: 'sfdc-contact-id',
+    defaultValue: '',
+  },
+  {
+    path: 'operator',
+    header: 'Operator',
+    name: 'operator',
+    defaultValue: 'no',
+    renderValue: value => (value ? 'yes' : 'no'),
+  },
+  {
+    path: 'accountId',
+    header: 'Acct ID',
+    name: 'acct-id',
+    defaultValue: '',
+  },
+]
+
+export const accountUserHeaderInfo = [
+  'First Name',
+  'Last Name',
+  'User Email',
+  'Quartz ID',
+  'IDPE ID',
+  'Onboarding Status',
+]
+
+export const acctUserColumnInfo: CellInfo[] = [
+  {
+    path: 'firstName',
+    name: 'first-name',
+    defaultValue: '',
+  },
+  {
+    path: 'lastName',
+    name: 'last-name',
+    defaultValue: '',
+  },
+  {
+    path: 'email',
+    name: 'user-email',
+    defaultValue: '',
+  },
+  {
+    path: 'quartzId',
+    name: 'user-quartz-id',
+    defaultValue: '',
+  },
+  {
+    path: 'id',
+    name: 'user-idpe-id',
+    defaultValue: '',
+  },
+  {
+    path: 'onboardingState',
+    name: 'onboarding-state',
+    defaultValue: '',
+  },
+]
+
+export const accountOrgHeaders = [
+  'Org Name',
+  'Org ID',
+  'Provider',
+  'Region',
+  'Balance',
+]
+
+export const acctOrgColumnInfo: CellInfo[] = [
+  {
+    path: 'name',
+    name: 'org-name',
+    defaultValue: '',
+  },
+  {
+    path: 'idpeID',
+    name: 'org-id',
+    defaultValue: '',
+    renderValue: value => (
+      <Link to={`/operator/organizations/${value}`}>{value}</Link>
+    ),
+  },
+  {path: 'provider', name: 'provider', defaultValue: ''},
+  {path: 'region', name: 'region', defaultValue: ''},
+  {
+    path: 'account.balance',
+    name: 'acct-balance',
+    defaultValue: '',
+  },
+]
+
+export const billingContactInfo: ResourceInfo[] = [
+  {
+    path: ['firstName', 'lastName'],
+    header: 'Name',
+    name: 'name',
+    defaultValue: '',
+    renderValue: value => {
+      return value.includes(undefined) ? 'N/A' : `${value[0]} ${value[1]}`
+    },
+  },
+  {
+    path: ['companyName'],
+    header: 'Company Name',
+    name: 'company-name',
+    defaultValue: '',
+  },
+  {
+    path: ['street1'],
+    header: 'Street 1',
+    name: 'street-one',
+    defaultValue: '',
+  },
+  {
+    path: ['street2'],
+    header: 'Street 2',
+    name: 'street-two',
+    defaultValue: '',
+  },
+  {
+    path: ['city', 'subdivision', 'postalCode'],
+    header: 'City State and Zip',
+    name: 'city-state-zip',
+    defaultValue: undefined,
+    renderValue: value => {
+      return value.includes(undefined) || value.includes(null)
+        ? 'N/A'
+        : `${value[0]}, ${value[1]} ${value[2]}`
+    },
+  },
+]
+
+export const OperatorRoutes = {
+  accounts: '/operator/accounts',
+  organizations: '/operator/organizations',
+  users: '/operator/users',
+}
```

Comment:
Are these used anywhere? As mentioned above, I think they should be because I think we want these as separate routes.